cfgov/v1/models/learn_page.py

```diff
@@ -105,6 +106,30 @@ class DocumentDetailPage(AbstractFilterPage):
 
     template = 'document-detail/index.html'
 
+    def related_metadata_tags(self, get_request):
+        # Set the tags to correct data format
+        tags = {'links': []}
+        # From the parent, get the form ids from the BrowseFilterablePage helper method
+        # then use the first id since the filterable list on the page will probably
+        # have the first id on the page. For more, see v1/models/browse_filterable_page.py line 105.
+        parent = self.parent()
+        form_ids = []
+        if callable(getattr(parent, 'get_form_specific_filter_data', None)):
+            form_ids = parent.get_form_specific_filter_data(parent.get_form_class(),
+                                                            get_request).keys()
+        if form_ids:
+            id = form_ids[0]
+        else:
+            id = None
```

Comment:
Can this section (up until `id = None`) be broken out into its own function? It could return the form_id, whether it is None or not. Mostly for readability purposes.